### PR TITLE
First version of README

### PR DESCRIPTION
**Pull request by kirill-ivanov-a**
- [x] first version of README added;
- [x] evaluation documentation and run examples added.

### CODE REVIEW

**Comment by kirill-ivanov-a** [comment]
#5 #12

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (anastasiia-kornilova) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Not sure that this is applicable, maybe better to make less overhead bug report template

File: README.md [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+---
+[![tests](https://github.com/prime-slam/evolin/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/prime-slam/evolin/actions/workflows/ci.yml)
+[![License: Apache License 2.0](https://img.shields.io/github/license/saltstack/salt)](https://opensource.org/license/apache-2-0/)
+
+EVOLIN is a benchmark for evaluation of line detection and association results. We provide a set of docker-packed line detection and association algorithms, metrics to evaluate them, and line-annotated data.
+Additional information can be found on our [web page](https://prime-slam.github.io/evolin/) and in the [article](https://arxiv.org/abs/2303.05162).
+
+## Installation
+1. Clone this repository.
+2. Install `g2opy` as described [here](https://github.com/uoip/g2opy).
```
Comment (anastasiia-kornilova) [inline README.md:14]: We use custom g2o from my repo, not original one

File: evaluation/association/classification/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  --output PATH, -o PATH
+                        output path (default: None)
+  --output-file STR, -O STR
+                        name of output file (default:
+                        classification_metrics.json)
+```
+
+### Example
+```bash
+python evaluate.py
```
Comment (anastasiia-kornilova) [inline evaluation/association/classification/README.md:35]: It seems that you need to add new line separator for every line, will copypasting into terminal work as it is now?

**Review by anastasiia-kornilova — CHANGES_REQUESTED** [review]

**Review by anastasiia-kornilova — APPROVED** [review]

**Pull request merged**